Day26-27/GlobalAssignmet.scilla

```diff
@@ -0,0 +1,77 @@
+(* Updated(upto part-4) on 31th March 2021 *)
+(* 0xe6e7163737fcf4782f6e348d753df4949dcb6b96 *)
+
+scilla_version 0
+
+library Miracle
+let responseStrDummyForNow1 = "response result1"
+let responseStrDummyForNow2 = "response result2"
+let reward = Uint128 0
```

Comment:
Change it to some positive value.
No user would invest without rewards :smile:

Reply:
Haha thats true, will make changes to it.
Thanks for the feedback :)